src/workspaces/NewWorkspaceModal/NewWorkspaceModal.test.ts

```diff
@@ -105,10 +105,12 @@ const setup = (opts: SetupOptions = {}): SetupResult => {
         Workspaces: {
           create: createWorkspace,
           workspace: () => ({
-            clone: cloneWorkspace,
```

Comment:
Can you please remove the clone method from the v1 Ajax endpoint? That will make it easier for future folks to understand what is used and mocked.